client/reader/following/main.jsx

```diff
@@ -8,28 +9,32 @@ import ReaderOnboarding from 'calypso/reader/onboarding';
 import SuggestionProvider from 'calypso/reader/search-stream/suggestion-provider';
 import Stream, { WIDE_DISPLAY_CUTOFF } from 'calypso/reader/stream';
 import ReaderListFollowedSites from 'calypso/reader/stream/reader-list-followed-sites';
-
+import Recent from '../recent';
 import './style.scss';
 
 function FollowingStream( { ...props } ) {
 	return (
 		<>
-			<Stream
-				{ ...props }
-				className="following"
-				streamSidebar={ () => <ReaderListFollowedSites path={ window.location.pathname } /> }
-			>
-				<BloganuaryHeader />
-				<NavigationHeader
-					title={ translate( 'Recent' ) }
-					subtitle={ translate( "Stay current with the blogs you've subscribed to." ) }
-					className={ clsx( 'following-stream-header', {
-						'reader-dual-column': props.width > WIDE_DISPLAY_CUTOFF,
-					} ) }
-				/>
+			{ config.isEnabled( 'reader/recent-feed-overhaul' ) ? (
```

Comment:
The only change here is that I'm checking the feature flag and rendering the new component if enabled.